--- libscrypt/lib/scryptenc/scryptenc.h~	2014-11-26 18:38:45.000000000 -0500
+++ libscrypt/lib/scryptenc/scryptenc.h	2014-11-26 18:39:02.000000000 -0500
@@ -80,6 +80,9 @@
  * 13	error reading input file
  */
 
+int pickparams(size_t, double, double,
+    int *, uint32_t *, uint32_t *);
+
 /**
  * scryptenc_buf(inbuf, inbuflen, outbuf, passwd, passwdlen,
  *     maxmem, maxmemfrac, maxtime):
--- libscrypt/lib/scryptenc/scryptenc.c~	2014-12-18 23:40:57.000000000 -0500
+++ libscrypt/lib/scryptenc/scryptenc.c	2014-12-18 23:41:04.000000000 -0500
@@ -48,12 +48,10 @@
 
 #define ENCBLOCK 65536
 
-static int pickparams(size_t, double, double,
-    int *, uint32_t *, uint32_t *);
 static int checkparams(size_t, double, double, int, uint32_t, uint32_t);
 static int getsalt(uint8_t[32]);
 
-static int
+int
 pickparams(size_t maxmem, double maxmemfrac, double maxtime,
     int * logN, uint32_t * r, uint32_t * p)
 {
